tools/read_data_files/README.md

```diff
@@ -0,0 +1,28 @@
+# read_data_files
+
+`read_data_files` is a utility to extract data for all timeseries' present in a TSDB file set.
+
+# Usage
+```
+$ git clone git@github.com:m3db/m3db.git
+$ make tools
```

Comment:
Might be nice to just specify the exact command for building this particular tool

Reply:
sure thing

Reply:
done